internal/blockchain/agendas_test.go

```diff
@@ -1017,6 +1017,117 @@ func TestSubsidySplitDeployment(t *testing.T) {
 	testSubsidySplitDeployment(t, chaincfg.RegNetParams())
 }
 
+// testBlake3PowDeployment ensures the deployment of the blake3 proof of work
+// agenda activates for the provided network parameters.
+func testBlake3PowDeployment(t *testing.T, params *chaincfg.Params) {
```

Comment:
This should also include logic to ensure the blake3 anchor block selection works as intended since I noticed there are not any tests currently hitting that path.

Reply:
Added.